policyengine_us/parameters/gov/states/ny/nysdps/clean_heat/incentives_structure_heating_capacity.yaml

```diff
@@ -0,0 +1,8 @@
+description: NYS Clean Heat program caps incentives at this amount for GSHP Full Load Heating projects for CHG&E, NGRID, NYSEG, O&R, RG&E.
```

Comment:
please add metadata if we want to make this a list parameter.

Example:
policyengine_us/parameters/gov/states/oh/tax/income/credits/non_refundable.yaml

Reply:
thank you! i changed my implementation approach, so this list would not be necessary anymore.